packages/test-utils/make-mount.ts

```diff
@@ -12,11 +11,11 @@ interface Options {
     [key: string]: any
   }
   methods?: {
-    [key: string]: (...args) => any
+    [key: string]: (...args: any[]) => any
```

Comment:
why not use `unknown`

Reply:
Updated. 

Sorry, I didn't think carefully here, because I think this file will be removed in the future.